google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITWriteTest.java

```diff
@@ -304,6 +304,22 @@ public void batchWriteAtLeastOnce() {
     }
   }
 
+  @Test
+  public void testWriteWithMaxCommitDelay() {
+    CommitResponse response =
+        client.writeWithOptions(
+            Collections.singletonList(
+                Mutation.newInsertOrUpdateBuilder("T")
+                    .set("K")
+                    .to(lastKey = uniqueString())
+                    .set("StringValue")
+                    .to("v1")
+                    .build()),
+            Options.maxCommitDelayInMilliSeconds(100));
```

Comment:
nit: In Go and Python we expose option `maxCommitDelay` directly to customers, though there is no guideline to align all the languages wanted to check if its intentional?

Reply:
We want to expose the maxCommitDelay directly to customers in all languages. Does this code not do this? I was certainly able to use these changes to change the spanner samples to exercise this option.

Reply:
I see `maxCommitDelayInMilliSeconds` option instead of `maxCommitDelay` being exposed here.

Reply:
Ah I see what you are saying - I will change it tomorrow.

Reply:
Done. My editor also drug in some formatting changes. Sorry...